### ISSUE TITLE
Memory usage

### ISSUE THREAD

**Issue by Strae** [lexcast/screen-recorder#1]
Just tryed to record a quick file edit, about 5 seconds, and I got out of ram before the gif could be saved.

The command 'convert' ate all my ram (8gb) and swapped for another couple of gigs.

Running Debian 8 64bit

**Comment by lexcast** [comment]
Yes, the issue is the ImageMagick convert command, I use a flag to optimize the output size, and that flag turns very slow this command when the recording is above 5 sec and a big area. But if I remove that flag, the output would be from ~4mb to ~25mb...
Maybe a configuration would help, like a Reduce Output?

**Comment by Strae** [comment]
Yes, a configuration option would be great.. and maybe another one to insert an optional throttle command for the imagemagick convert: who's under linux could use cpulimit for example (dunno about ios/windows)

**Comment by Strae** [comment]
p.s: anyway thanks man, this is a nice plugin!

**Comment by lexcast** [comment]
Thanks, I will work on that config option when I can.
The throttle command is new for me, a link would be helpful :smile:

**Comment by Strae** [comment]
https://github.com/opsengine/cpulimit
http://cpulimit.sourceforge.net/

**Comment by lexcast** [comment]
Thanks, I'll check it out

**Comment by lexcast** [comment]
I've put some config options for fps and reduceOutput. 

For the cpu usage, ImageMagick uses some env variables http://www.imagemagick.org/script/resources.php#environment 
Maybe the MAGICK_THROTTLE variable could work.
